core/src/test/scala/org/genivi/sota/core/DeviceUpdatesResourceSpec.scala

```diff
@@ -73,6 +74,21 @@ class DeviceUpdatesResourceSpec extends FunSuite
     }
   }
 
+  test("system_info updates are forwarded to device registry") {
+    import SimpleJsonGenerator._
+
+    val json1 = simpleJsonGen.sample.get
```

Comment:
we could name this better, like `newSystemInfo`  and `updatedSystemInfo` or sth?